measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
```

Comment:
@akegan thanks - at the moment I've limited options to 'claims' and 'registry'. I think @marimiyachi is wondering why we're limiting it to these 2 methods (there are other methods like EHR) but my understanding is those are the only two submission methods that these code specifications apply to. I guess another way to phrase this might be only registries and claims are measuring and reporting these measures using these clinical coding standards. Is that understanding correct?